website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -4,35 +4,39 @@ short: Helm
 weight: 3
 ---
 
-[Helm] is a package manager for Kubernetes that facilitates the deployment and management of applications and services on Kubernetes clusters. This page covers installing and managing Vector through the Helm package repository.
+[Helm] is a package manager for Kubernetes that facilitates the deployment and management of applications and services on Kubernetes clusters. This page covers installing and managing the Vector Agent and the Vector Aggregator from the Helm package repository. The Agent and the Aggregator are added by adding the Vector Helm chart but are installed and configured separately.
 
-## Installation
+{{< warning title="Aggregator role in public beta" >}}
+Helm support for the [aggregator] role is currently in public beta. We're currently seeking beta testers. If interested, please [join our chat][chat] and let us know.
 
-{{< warning title="Aggregator role in private beta" >}}
-Helm support for the [aggregator] role is currently in private beta. We're currently seeking beta testers. If interested, please [join our chat][chat] and let us know.
-
-As an alternative, you can still manually deploy Vector in the aggregator role. Instructions throughout this doc will be for the [agent] role only.
-
-[agent]: /docs/setup/deployment/roles/#agent
 [aggregator]: /docs/setup/deployment/roles/#aggregator
 [chat]: https://chat.vector.dev
 {{< /warning >}}
 
-Add the Vector repo:
+## Adding the Helm repo
+
+If you haven't already, start by adding the Vector repo:
 
 ```shell
-helm repo add vector https://helm.vector.dev
+helm repo add timberio https://packages.timber.io/helm/latest
```

Comment:
Is this right, shouldn't it be `helm repo add vector ...` cc/ @spencergilbert

Reply:
Both are supported for this release - we should probably make sure it's consistent between this page and the highlight

Reply:
👍 do you have a preference? I'm inclined to use the new repo since these are new docs and they assume you are starting from scratch.

Reply:
Sure - we can keep the `helm repo add vector https://helm.vector.dev` and sed timberio/vector